modules/ROOT/pages/import-mule-app-project.adoc

```diff
@@ -12,6 +12,15 @@ You can import a Mule-application project that was exported from Design Center.
 * Importing a project that was exported from a different environment within a single organization
 * Importing a project that was exported from an environment that is in a different organization
 
+[IMPORTANT]
+====
+An exported project does not include items that the project linked to in its original location.
```

Comment:
Actually when exporting a project, the pom inside the jar has the references to its external dependencies. When importing the project, the external dependencies in the pom will me omitted.

Reply:
I changed "exported" to "imported". I've also made a number of other related changes.